onyo/commands/unset.py

```diff
@@ -69,4 +69,4 @@ def unset(args: argparse.Namespace) -> None:
               args.filter,
               args.dry_run,
               args.depth,
-              args.message)
+              message='\n\n'.join(m for m in args.message) if args.message else None)
```

Comment:
Pyre type-checking fails here.

Reply:
A dang, yes. The change in the other function is waiting for a merge of #422